### DISCUSSION TITLE
Grid

### DISCUSSION THREAD

**Discussion by vonhex** [ausbitbank/stable-diffusion-discord-bot#28] [Ideas]
I know Invokeai can use --grid from within the CLI, is there a way we can get gridded outputs with separate upscale buttons and such like Midjourney? Maybe also a live render percentage or visual?

**Comment by ausbitbank** [comment]
As far as I can tell you can't use the grid option for the current web api that the bot uses, there is an impending major api change coming for invokeai that could change that though.

Upscale buttons are already there inside the tweak menu so you could just request 4 images at a time for each prompt.

Live render previews aren't really practical with the current api setup via discord bot, I don't know what sorcery midjourney is using to be allowed to update the image preview like they do. Maybe special access as a discord partner ?

**Reply by vonhex** [reply]
hmm gotcha! I do have another issue if you have time.

**Reply by ausbitbank** [reply]
Sure, maybe just drop into the discord https://discord.gg/tYJuU2ueZ4

**Comment by vonhex** [comment]
Well done on the live preview!
